examples/cpp/dds/CustomPayloadPoolExample/CustomPayloadPool_main.cpp

```diff
@@ -209,7 +149,7 @@ int main(
         }
 
         // For backward compatibility count and sleep may be given positionally
-        if (parse.nonOptionsCount() > 3 || parse.nonOptionsCount() == 0)
+        if (parse.nonOptionsCount() > 2 || parse.nonOptionsCount() == 0)
         {
             throw 2;
         }
```

Comment:
This bit is there in the helloworld for compatibility with previous CLI, remove it here